src/portable/chipidea/ci_hs/hcd_ci_hs.c

```diff
@@ -50,6 +50,10 @@
     SCB_InvalidateDCache_by_Addr(addr, (int32_t) data_size);
   }
 
+void hcd_dcache_clean_invalidate(void* addr, uint32_t data_size) {
```

Comment:
How about make the cache related function optional (default ON) ?

For IMX.RT in order to have best performance, RAM will be configured to ITCM & DTCM, so they are accessed at CPU clock instead of bus clock (600MHz vs 133MHz)

Since TCM bypass the cache there is no need to flush or invalidate.

![image](https://github.com/hathach/tinyusb/assets/4375114/0262587f-a4a1-4ab3-b659-6aab59af8dc6)

Reply:
ah you are right, I think we can check the address to see if it is within the DTCM (non-cachable) and skip the clean/invalidate will update in the next push or so. Circuitpython is currently placing ehci_data and/or user memory randomly and may not be in DTCM.

Reply:
DTCM is 1MB reserved in 2000000-2010000,this should be good enough.

![image](https://github.com/hathach/tinyusb/assets/249515/439390a4-7be1-4e47-b8e6-45e702d8601f)